.changeset/quick-cycles-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-zod-client": major
```

Comment:
could you change that to "patch" ? major would update the lib's version to 2
patch will update to 1.6.3
(and minor would update to 1.7)

Reply:
my bad, changed to `patch`